setup.py

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
```

Comment:
Please follow the style of `setup.py` used by the rest of our other sdks
https://github.com/wavefrontHQ/wavefront-sdk-python/blob/master/setup.py
https://github.com/wavefrontHQ/wavefront-pyformance/blob/master/setup.py
https://github.com/wavefrontHQ/wavefront-opentracing-sdk-python/blob/master/setup.py